packages/strapi/lib/configuration/hooks/dictionary/_plugins/index.js

```diff
@@ -129,10 +129,12 @@ module.exports = strapi => {
             controllers: plugin['controllers/*'],
             models: plugin['models/*'],
             services: plugin['services/*'],
-            policies: plugin['policies/*'],
             config: plugin['config/**']
           };
 
+          // Set policies value.
+          strapi.plugins[plugin.name].config.policies = plugin['config/policies/*'];
```

Comment:
Add a condition to only set the `config.policies` object when `plugins['config/policies/*]` is not empty to avoid to make the next line obsolete.